DependencyInjection/SpClient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Italia\SpidSymfonyBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerInterface as Container;
+use Italia\Spid\Sp;
+
+class SpClient
+{
+
+    private $spClientConfigured;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(Container $container)
+    {
+
+        $configuration = [
+            'sp_entityid' => $container->getParameter('spid.sp_entityid'),
+            'sp_key_file' => $container->getParameter('spid.sp_key_file'),
+            'sp_cert_file' => $container->getParameter('spid.sp_cert_file'),
+            'sp_assertionconsumerservice' => $container->getParameter('spid.sp_assertionconsumerservice'),
+            'sp_singlelogoutservice' => $container->getParameter('spid.sp_singlelogoutservice'),
+            'sp_org_name' => $container->getParameter('spid.sp_org_name'),
+            'sp_org_display_name' => $container->getParameter('spid.sp_org_display_name'),
+            'idp_metadata_folder' => $container->getParameter('spid.idp_metadata_folder') . DIRECTORY_SEPARATOR,
+            'sp_attributeconsumingservice' => $container->getParameter('spid.sp_attributeconsumingservice'),
+        ];
+        $this->spClientConfigured = new Sp($configuration);
+    }
+    public function getSpClientConfigurated()
```

Comment:
If we stick to English for function names let's use correct constructions:
`getConfiguredSPClient`